elcid/models.py

```diff
@@ -758,6 +782,47 @@ def get_api_name(cls):
         return camelcase_to_underscore(cls._meta.object_name)
 
 
+class ChronicAntifungal(omodels.PatientSubrecord):
+    DRUG_LIST = "drug_list"
```

Comment:
I think drug list is ambiguous here - first pass I read this as 'reason == something to do with a list of drugs' whereas it's the dispensary report rather than something like the list of `    ANTIFUNGAL_DRUGS = [ ` above?